docs_src/css/designsystem.css

```diff
@@ -1018,56 +1019,33 @@ a.badge:hover {
   text-decoration: none;
   white-space: nowrap;
   height: var(--button-height);
-}
-
-/* Default button coloring */
-
-.button:not([class*="bg--"]) {
   background: var(--color--white);
-  border: 1px solid var(--color--section, var(--color--default));
-  color: var(--color--section, var(--color--default));
-}
-
-/* Custom button - temp solution */
-
-.button[class*="bg--"] {
-  border: 1px solid transparent;
-}
-
-/* On hover */
-
-.button[class*="bg--"]:active,
-.button[class*="bg--"]:hover,
-.button[class*="bg--"]:focus {
-  background: var(--color--white);
-  border: 1px solid var(--color--graa5);
-  color: var(--color--black);
+  border: 1px solid var(--button-color);
+  color: var(--button-text-color, var(--button-color));
 }
 
 .button:disabled {
   cursor: default;
-  opacity: .7;
+  filter: grayscale(100%);
```

Comment:
Kan vi bare sætte den til background: var(--color--graa6) + color: var(--color--graa3), for at undgå "hack"?

Reply:
det var dit eget forslag :-P
men jo det er nu button-color som på Variations